src/bundle/Resources/public/scss/_extra-actions.scss

```diff
@@ -162,8 +162,30 @@
             }
         }
     }
+}
+
+.ez-extra-actions-container {
+    .ez-extra-actions {
+        position: absolute;
+        top: calculateRem(-24px);
+        bottom: 0;
+        left: 0;
+        z-index: 2;
+        opacity: 1;
+        transform: translate(calc(-100%), 0) scaleX(1);
+        transform-origin: right center;
+        transition: all 0.4s ease-in;
 
-    &--dropdown {
+        &--hidden {
+            opacity: 0;
+            transform: translate(calc(-100%), 0) scaleX(0);
+            transition: all 0.2s ease-out;
+        }
+    }
+}
+
+.ez-dropdown {
```

Comment:
admin-ui doesn't know about page-builder. This cannot be style here. Should be in page-builder.